src/components/Comments.jsx

```diff
@@ -1,4 +1,33 @@
+import { getComments } from "../utils/api";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
 const Comments = () => {
-  return <h1>This is a Comments</h1>;
+  const params = useParams();
+  const id = params.article_id;
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    getComments(id).then((allComments) => {
+      setComments(allComments);
+    });
+  }, [id]);
+
+  return (
+    <section className="Article">
+      <h1>Comments:</h1>
+      {comments.map((comment) => {
```

Comment:
What do you want to render if an article has no comments? User would need some feedback so they know that their request has been sent but nothing is there yet.